pylint/checkers/base/comparison_checker.py

```diff
@@ -212,6 +220,22 @@ def _check_logical_tautology(self, node: nodes.Compare):
             suggestion = f"{left_operand} {operator} {right_operand}"
             self.add_message("comparison-with-itself", node=node, args=(suggestion,))
 
+    def _check_two_literals_being_compared(self, node: nodes.Compare) -> None:
+        """Check if two literals are being compared; this is always a logical tautology."""
+        left_operand = node.left
+        right_operand = node.ops[0][1]
+        operator = node.ops[0][0]
+        if not isinstance(left_operand, nodes.Const):
+            return
+        if not isinstance(right_operand, nodes.Const):
+            return
```

Comment:
```suggestion
        left_operand = node.left
        if not isinstance(left_operand, nodes.Const):
            return

        right_operand = node.ops[0][1]
        if not isinstance(right_operand, nodes.Const):
            return

        operator = node.ops[0][0]
```

A bit nitpicky, but we can save some assignments if the `if` can fail earlier.

Reply:
Your changed code is more performant and should behave exactly the same